superset-frontend/src/pages/Login/index.tsx

```diff
@@ -74,7 +74,62 @@ const StyledLabel = styled(Typography.Text)`
   `}
 `;
 
+const StyledBackground = styled.div`
+  ${({ theme }) => {
+    const bgImageUrl = theme.loginPageBackgroundImageUrl;
+    const overlayColor =
+      theme.loginPageBackgroundOverlayColor || 'rgba(0, 0, 0, 0.5)';
+    const blurAmount = theme.loginPageBackgroundBlur || '0px';
+    return bgImageUrl
+      ? css`
+          position: fixed;
+          top: 0;
+          left: 0;
+          width: 100vw;
+          height: 100vh;
+          z-index: 0;
+          pointer-events: none;
+
+          &::before {
+            content: '';
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+            background-image: url(${bgImageUrl});
+            background-size: cover;
+            background-position: center;
+            filter: blur(${blurAmount});
+            z-index: -1;
+          }
+
+          &::after {
+            content: '';
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+            background-color: ${overlayColor};
+            z-index: 1;
+          }
```

Comment:
Are these psuedo element styles necessary? I think we can simplify this a bit further. Something like this with more tweaks:

```suggestion
					background: ${ bgImageUrl ? url(bgImageUrl) : overlayColor }
```